src/views/activity/components/transaction-card/transaction-card.view.tsx

```diff
@@ -4,18 +4,20 @@ import useTransactionCardStyles from "src/views/activity/components/transaction-
 import { ReactComponent as TransferL1Icon } from "src/assets/icons/l1-transfer.svg";
 import { ReactComponent as TransferL2Icon } from "src/assets/icons/l2-transfer.svg";
 import { ReactComponent as ReloadIcon } from "src/assets/icons/spinner.svg";
-import { ReactComponent as EthToken } from "src/assets/tokens/eth.svg";
-import { ReactComponent as DaiToken } from "src/assets/tokens/dai.svg";
 import Typography from "src/views/shared/typography/typography.view";
 import Card from "src/views/shared/card/card.view";
 import { getTimeFromNow, TimeFromNowParams } from "src/utils/time";
 import { convertTokenAmountToFiat } from "src/utils/amounts";
+import { useNavigate } from "react-router-dom";
+import routes from "src/routes";
+import TokenIcon from "src/views/shared/token-icon/token-icon";
 
-enum StatusText {
+export enum StatusText {
```

Comment:
As now this enum is shared I would write it in the `domain` module, next to the `WalletName`. Also, remember that enum properties should be in MAYUS and with underscores:

```ts
enum StatusText {
  ON_HOLD = "On Hold",
  INITIATED = "Initiated",
  PROCESSING = "Processing",
  FAILED = "Error",
  COMPLETED = "Completed"
}
```

Reply:
This is waiting to receive data from API so properties maybe can't be in uppercase

Reply:
But enum properties can not be in lowercase, if they should be like this it should be a data structure but not an enum

Reply:
I've changed it to object in that case